### ISSUE TITLE
Is 'llava.train' original?

### ISSUE THREAD

**Issue by HovChen** [DCDmllm/HealthGPT#5]
Are you using the original 'lava.train'? Or maybe you changed something?

**Comment by Lin-Tianwei** [comment]
We add multimodal tokens to the vocabulary, and it's necessary to appropriately turn the gradients of different modules on and off according to different training stages. But overall, yes, our training code is logically based on `llava.train`.
